monitoring/ceph-mixin/dashboards_out/smb-overview.json

```diff
@@ -199,27 +209,32 @@
             "y": 0
          },
          "id": 4,
-         "links": [ ],
          "options": {
             "colorMode": "none",
             "graphMode": "none",
             "justifyMode": "auto",
             "orientation": "auto",
+            "percentChangeColorMode": "standard",
             "reduceOptions": {
                "calcs": [
                   "lastNotNull"
                ],
                "fields": "",
                "values": false
             },
-            "textMode": "auto"
+            "showPercentChange": false,
+            "textMode": "auto",
+            "wideLayout": true
          },
-         "pluginVersion": "9.4.7",
+         "pluginVersion": "11.6.0",
          "targets": [
             {
-               "datasource": "${datasource}",
+               "datasource": {
+                  "uid": "${datasource}"
+               },
+               "editorMode": "code",
                "exemplar": false,
-               "expr": "sum by(netbiosname)(smb_sessions_total * on (instance) group_left (netbiosname) smb_metrics_status{netbiosname=~\"$SMBcluster\"}) / (count by ( netbiosname)(smb_sessions_total * on (instance) group_left (netbiosname) smb_metrics_status{netbiosname=~\"$SMBcluster\"}))",
+               "expr": "sum(sum by(netbiosname)(smb_sessions_total * on (instance) group_left (netbiosname) smb_metrics_status{netbiosname=~\"$SMBcluster\"}) / (count by ( netbiosname)(smb_sessions_total * on (instance) group_left (netbiosname) smb_metrics_status{netbiosname=~\"$SMBcluster\"})))",
```

Comment:
Same here! I guess it should be `ceph_smb_metadata` and not `smb_metrics_status`

Reply:
https://github.com/ceph/ceph/pull/64719#discussion_r2239735566